crime-analytics.ipynb

```diff
@@ -60,10 +60,8 @@
   },
```

Comment:
<p>Example PR done with ReviewNB @chuaner - you will probably need to create an account on ReviewNB to see it, and once logged in you should be able to access the PR in ReviewNB.</p>

 _Reply via <a href='https://app.reviewnb.com/cmenguy/crime-analytics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>